src/test/java/ru/nmedvedev/service/spendmoneyreminder/RemindDayProviderServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package ru.nmedvedev.service.spendmoneyreminder;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+class RemindDayProviderServiceTest {
+
+    private RemindDayProviderService remindDayProviderService = new RemindDayProviderService();
+
+    @Test
+    void returnHalfOfTheMonth() {
+        assertThat(remindDayProviderService.getDay(LocalDate.of(2020, 1, 15)), is(ReminderDayEnum.HALF_MONTH));
+        assertThat(remindDayProviderService.getDay(LocalDate.of(2020, 1, 14)), is(ReminderDayEnum.NOT_A_DAY_FOR_A_REMINDER));
```

Comment:
I guess one test method shoud contain only one target method invocation. You can vary arguments using [parameterized tests](https://junit.org/junit5/docs/current/user-guide/#writing-tests-parameterized-tests)

Reply:
I can vary only one argument as far as I know. In tests here I need to vary at least 3.

Reply:
No, you can vary as many arguments as you wish